src/tools/rbd/action/Bench.cc

```diff
@@ -16,13 +16,21 @@
 
 namespace rbd {
 namespace action {
-namespace bench_write {
+namespace bench {
 
 namespace at = argument_types;
 namespace po = boost::program_options;
 
 namespace {
 
+enum io_type_t {
+  BENCH_READ = 0,
+  BENCH_WRITE,
+
+  BENCH_LAST,
+};
```

Comment:
Also, you could consider using names IO_TYPE_READ, IO_TYPE_WRITE... as it has been suggested by Jason, to follow convention.